presto-main/src/main/java/io/prestosql/security/AccessControlManager.java

```diff
@@ -24,6 +24,7 @@
 import io.prestosql.metadata.QualifiedObjectName;
```

Comment:
I'd rephrase the commit as "Improve robustness of InitializeSystemAccessControl", and in the commit message indicate that this makes it easier to evolve the SystemAccessControl interface by adding new methods.